integrations/google_ai/src/haystack_integrations/components/generators/google_ai/chat/gemini.py

```diff
@@ -2,9 +2,10 @@
 from typing import Any, Dict, List, Optional, Union
 
 import google.generativeai as genai
-from google.ai.generativelanguage import Content, Part, Tool
+from google.ai.generativelanguage import Content, Part
+from google.ai.generativelanguage import Tool as ToolProto
```

Comment:
Combine the two?

Reply:
Can't do, `ruff` doesn't like that and I would agree.